WordPress/Classes/Utility/WPImmuTableRows.swift

```diff
@@ -236,6 +236,34 @@ struct LinkWithValueRow: ImmuTableRow {
     }
 }
 
+/// Create a row button: uses the WordPress branded blue and is left aligned
+///
+struct BrandedButtonRow: ImmuTableRow {
+    static let cell = ImmuTableCell.class(WPTableViewCellIndicator.self)
+
+    let title: String
+    let showIndicator: Bool
+    let action: ImmuTableAction?
+    let accessibilityIdentifier: String?
+
+    init(title: String, action: @escaping ImmuTableAction, showIndicator: Bool = false, accessibilityIdentifier: String? = nil) {
+        self.title = title
+        self.showIndicator = showIndicator
+        self.action = action
+        self.accessibilityIdentifier = accessibilityIdentifier
+    }
+
+    func configureCell(_ cell: UITableViewCell) {
+        let cell = cell as! WPTableViewCellIndicator
```

Comment:
since this is an internal method that can be called from anywhere in the code, we do not seem to have a full guarantee that the passed type will actually be `WPTableViewCellIndicator`, so I think we should remove the force unwrap in favor of a `guard` statement.